vlttng/venv.py

```diff
@@ -547,14 +547,18 @@ def _build_lttng_tools(self):
         if project is None:
             return
 
-        if '--with-lttng-ust-prefix' in project.configure or '--without-lttng-ust' in project.configure:
-            msg = 'I would not pass the --with-lttng-ust-prefix/--without-lttng-ust configure option if I were you: they are handled by vlttng'
-            _pwarn(msg)
+        for ust_config in ['--with-lttng-ust-prefix', '--without-lttng-ust', '--disable-lttng-ust']:
+            if ust_config in project.configure:
+                msg = 'I would not pass the '+ust_config+' configure option if I were you: they are handled by vlttng'
```

Comment:
Yes you're right, no join then.

Also, prefer using a tuple than a list for `lttng_ust_opts`.